src/vendor/jstyleson.py

```diff
@@ -0,0 +1,137 @@
+# From https://github.com/linjackson78/jstyleson/blob/8c47cc9e665b3b1744cccfaa7a650de5f3c575dd/jstyleson.py
```

Comment:
Vendoring this one because it's painful to pip install stuff.  Small file so I don't think it's a big deal.  We use this for parsing jsonc